frame/support/procedural/src/construct_runtime/expand/config.rs

```diff
@@ -74,12 +74,18 @@ pub fn expand_outer_config(
 		#[serde(rename_all = "camelCase")]
 		#[serde(deny_unknown_fields)]
 		#[serde(crate = "__genesis_config_serde_import__")]
-		pub struct GenesisConfig {
+		pub struct RuntimeGenesisConfig {
 			#fields
 		}
 
+		// TODO [#14065]
+		// - deprecate it
```

Comment:
Could you maybe temporarily put a deprecation here, to see that the CI stays green and we got all cases?

Reply:
Sure, that's what I did locally. Let me know once you confirm it yourself so I can remove it

Reply:
Yes you can do them lean without companion (since that is annoying).